lib/athena_health/version.rb

```diff
@@ -1,3 +1,3 @@
 module AthenaHealth
-  VERSION = '2.0.4'.freeze
+  VERSION = '2.0.6'.freeze
```

Comment:
Just curious, did we mean to skip 2.0.5?

Reply:
I pushed up 2.0.5 when I cloned it.